contract_commission/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+Commission in contracts
+=======================
+
+This module allows to get default commissions for partner in the recurring
+invoices generated from contracts.
+
+Installation
+============
+
+This module is auto-installed when you have *account_analytic_analysis* and
+*sale_commission* modules installed.
+
+Usage
+=====
+
+When invoicing contracts, now agents assigned to the partner of the contract
+will be assigned on the generated invoices
```

Comment:
When invoicing contracts, the agents assigned to the partner of the contract
will now be assigned to the generated invoices